### DISCUSSION TITLE
swa with nuxt

### DISCUSSION THREAD

**Discussion by sig9** [Azure/static-web-apps-cli#442] [Q&A]
I am running into an issue where I see 405 errors when I run swa in front of nuxt using the content module. 

With silly debugging turned on I see the method POST is not allowed to a path starting with /_content/ . Presumably because this does not start with /api. 

Can there be more than one apiPrefix for a dev server setup?

I've experimented unsuccessfully with adding a route in staticwebapp.config.json.
    {
      "route": "/_content/*",
      "allowedRoles": ["anonymous","authenticated"]
    }

**Comment by manekinekko** [comment]
Hey @sig9 can you report this in our issue tracker. We will investigate it. Thank you.

**Reply by manekinekko** [reply]
No worries, I've moved this discussion to https://github.com/Azure/static-web-apps-cli/issues/461
